websites/W/WebProfiles/presence.ts

```diff
@@ -4,25 +4,39 @@ const presence = new Presence({
 	browsingTimestamp = Math.floor(Date.now() / 1000);
 
 presence.on("UpdateData", async () => {
+	let presenceData: PresenceData = {
+		largeImageKey: "webprofiles_logo",
```

Comment:
Imgur links are preferred over Discord asset names.
```suggestion
		largeImageKey: "https://i.imgur.com/dPN4dbK.png",
```